scout/constants/__init__.py

```diff
@@ -90,6 +90,10 @@
     VERIFIED_VARIANTS_HEADER,
 )
 
+# Default number of threads for variant loading
+LOADER_THREADS = 8
```

Comment:
They are mostly there as defaults, and would presumably just be aliased in, but it would arguably be nice to be able to set them in config as well. In that case we better use the old CLI-one for now. I'll add a suggestion!